oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/BackChannelLogoutRequestHandler.java

```diff
@@ -81,6 +84,7 @@ public JWTClaimsSet generateClaims(BackChannelLogoutMessage inputEvent) {
                 .jwtID(UUID.randomUUID().toString())
                 .audience(inputEvent.getClientId())
                 .subject(inputEvent.getSubjectId())
+                .expirationTime(Date.from(clock.instant().plus(2, ChronoUnit.MINUTES)))
```

Comment:
Is it better to use the `NowHelper.java` here

Reply:
I thought that, but it's difficult to stub that out. Maybe that's a refactoring smell itself. 🤔

I'll timebox a bit to see if I can make the NowHelper more testable

Reply:
Yes, good point. I'm happy to approve if this is something we want to come back to.